pkg/bgp/speaker/speaker.go

```diff
@@ -34,12 +36,16 @@ import (
 	"k8s.io/client-go/util/workqueue"
 )
 
+var (
+	ErrShutDown = errors.New("cannot enqueue event, speaker is shutdown")
```

Comment:
Does this need to be exported?

Reply:
It would allow the k8s watcher to determine if it wants to react to the bgp speaker being shut down.